pkg/generic/descriptor/http_mapping.go

```diff
@@ -68,7 +68,7 @@ var NewAPIQuery NewHTTPMapping = func(value string) HTTPMapping {
 }
 
 func (m *apiQuery) Request(ctx context.Context, req *HTTPRequest, field *FieldDescriptor) (interface{}, bool, error) {
-	val := req.Query.Get(m.value)
+	val := req.GetQuery(m.value)
```

Comment:
do check on the possible performance issue related to req.Query

Reply:
made cache for query.